src/numbers.js

```diff
@@ -1,45 +1,49 @@
 function add (a, b) {
-  // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
-  // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
-  // your code here
+  return a * b;
 }
 
 function divide (a, b) {
-  // your code here
+  return a / b;
 }
 
 function power (a, b) {
-  // your code here
+  return a ** b;
 }
 
 function round (a) {
-  // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
-  // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
-  // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
-  // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
-  // your code here
+  if (Math.sign(a/b) === -1){
```

Comment:
I have never seen or used Math.sign before, so you have taught me something new today! Thanks!

Reply:
See if you can refactor this using a ternary statement. :)